db/blip_sync_stats.go

```diff
@@ -141,6 +141,7 @@ func BlipSyncStatsForCBL(dbStats *base.DbStats) *BlipSyncStats {
 	blipStats.HandleChangesResponseCount = dbStats.CBLReplicationPull().RequestChangesCount
 	blipStats.HandleChangesResponseTime = dbStats.CBLReplicationPull().RequestChangesTime
 	blipStats.HandleChangesSendRevCount = dbStats.CBLReplicationPull().RevSendCount
+	blipStats.SendRevErrorTotal = dbStats.CBLReplicationPull().RevErrorCount
```

Comment:
For consistency, can we move this up with the other SendRev stats (after line 131)?  I realize there's some ambiguity but it looks like the `SendRev*` stats are the ones that are set in sendRevision, and the `HandleChanges*` stats are incremented in handleChangesResponse.